packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.controller.js

```diff
@@ -0,0 +1,88 @@
+import { COMFORT_EXCHANGE_TYPE_ERROR } from './pack-xdsl-access-comfort-exchange.constant';
+
+export default class XdslAccessComfortExchangeCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    OvhApiXdsl,
+    TucToast,
+    TucToastError,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.OvhApiXdsl = OvhApiXdsl;
+    this.TucToast = TucToast;
+    this.TucToastError = TucToastError;
+  }
+
+  $onInit() {
+    this.exchange = {
+      order: {
+        url: null,
+        id: null,
+        priceWithoutTax: null,
+        priceWithTax: null,
+        tax: null,
+      },
+      isSuccess: false,
+    };
+    this.isAvailable = false;
+    this.getListOpenedRMA();
+  }
+
+  getListOpenedRMA() {
+    this.isRMAOpened = false;
+    return this.OvhApiXdsl.RMA()
+      .v6()
+      .query({
+        xdslId: this.xdslId,
+      })
+      .$promise.then((rmas) => {
+        if (rmas.length > 0) {
+          this.rmas = [];
+          this.isRMAOpened = true;
+          rmas.forEach(id => this.OvhApiXdsl.RMA()
+            .v6()
+            .get(
+              {
+                xdslId: this.xdslId,
+              },
+              { id },
+            )
+            .$promise.then((rma) => {
+              const addRma = {
```

Comment:
I guess `rma` have more properties than you need to return in a new variable ? If no you could push directly `rma`

Reply:
yes it is. But I don't need to display all properties.